pkg/agent/config.go

```diff
@@ -69,4 +69,6 @@ type Config struct {
 	KafkaCompression string `env:"KAFKA_COMPRESSION" envDefault:"none"`
 	// ProfilePort sets the listening port for Go's Pprof tool. If it is not set, profile is disabled
 	ProfilePort int `env:"PROFILE_PORT"`
+	// EvictionTimeout sets the timeout for eviction of an inactive flow, Default is 2 seconds
+	EvictionTimeout uint64 `env:"EVICTION_TIMEOUT" envDefault:"2000000000"`
```

Comment:
Configuration values in nanoseconds are very unfriendly to the users. I did this PR against your own PR to use durations:

https://github.com/netobserv/netobserv-ebpf-agent/pull/45/files

Reply:
Thanks for adding this.